src/components/DataViewer/DataViewer.tsx

```diff
@@ -15,45 +17,70 @@ const StyledModal = styled(Modal)`
     left: 50%;
     transform: translate(-50%, -50%);
     border: 0;
-    border-radius: 5px;
+    border-radius: 2em;
     outline: none;
     padding: 0;
     width: 80%;
     max-height: 80%;
-    background: rgb(255, 255, 255);
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
     box-shadow: rgb(200, 200, 200) 0 2px 10px;
     overflow: hidden;
+    max-width: 1000px;
 `;
-const Header = styled.h3`
+
+const Header = styled.div`
     display: flex;
     justify-content: space-between;
     align-items: center;
-    background-color: #eeeeee;
-    padding: 0.5rem;
-    margin-top: 0;
+    color: ${(props) => props.theme.secondaryForegroundColor};
+    background-color: ${(props) => props.theme.secondaryBackgroundColor}A0;
+    font-size: 1.5em;
+    font-weight: bolder;
+    padding: 0.75em 1em 0.75em 1em;
```

Comment:
I think this is the same as `padding: 0.75em 1em` ?

Reply:
Refactored